xbmc/cores/VideoPlayer/DVDCodecs/Overlay/DVDOverlaySSA.h

```diff
@@ -34,7 +34,6 @@ class CDVDOverlaySSA : public CDVDOverlay
   {
     replace = true;
     m_libass = libass;
-    libass->Acquire();
```

Comment:
@FernetMenta the object created here: https://github.com/xbmc/xbmc/blob/master/xbmc/cores/VideoPlayer/DVDSubtitles/DVDSubtitleParserSSA.cpp#L56

never gets deleted. This results in the refcount of libass not reaching zero, and hence the libass memory not being freed.